src/mcts/params.cc

```diff
@@ -247,11 +251,13 @@ void SearchParams::Populate(OptionsParser* options) {
   options->HideOption(kNoiseEpsilonId);
   options->HideOption(kNoiseAlphaId);
   options->HideOption(kLogLiveStatsId);
+  options->HideOption(kOneMinusEpsId);
 }
 
 SearchParams::SearchParams(const OptionsDict& options)
     : options_(options),
       kLogitQ(options.Get<bool>(kLogitQId.GetId())),
+      kOneMinusEps(options.Get<float>(kOneMinusEpsId.GetId())),
```

Comment:
As stated above it may be better to multiply by 0.99999994f here instead of having it as default value.